runtime/interpreter/big.go

```diff
@@ -140,3 +141,31 @@ func BigEndianBytesToSignedBigInt(b []byte) *big.Int {
 func BigEndianBytesToUnsignedBigInt(b []byte) *big.Int {
 	return new(big.Int).SetBytes(b)
 }
+
+func BigIntSqrt(interpreter *Interpreter, value *big.Int, locationRange LocationRange) UFix64Value {
+	if value.Sign() < 0 {
+		panic(UnderflowError{
+			LocationRange: locationRange,
+		})
+	}
+
+	valueFloat := new(big.Float).SetPrec(256).SetInt(value)
+	res := new(big.Float).SetPrec(256).SetMode(big.ToZero).Sqrt(valueFloat)
+
+	valueGetter := func() uint64 {
+		res.Mul(res, new(big.Float).SetPrec(256).SetInt(sema.Fix64FactorBig))
+
+		resInt := new(big.Int)
+		res.Int(resInt)
```

Comment:
why are we computing this in the getter (presumably multiple times if the value is accessed multiple times)? Can we can we move that outside of `valueGetter`?

Reply:
Done in 405fed080